k8s/knative/chart/knative/templates/knative/eventing/controller.yaml

```diff
@@ -0,0 +1,98 @@
+# Copyright 2018 The Knative Authors
```

Comment:
Use Google headers and 2022. Here and everywhere

Reply:
Done